pkgs/by-name/or/orca/package.nix

```diff
@@ -56,6 +60,12 @@ python3.pkgs.buildPythonApplication rec {
     yelp-tools
     itstool
     gobject-introspection
+  ]
+  ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
```

Comment:
Is there any harm adding those all the time to make the builds more similar between native and cross?

Reply:
fixed.